.solcover.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
     norpc: true,
     port: 8545,
     copyPackages: ['openzeppelin-solidity'],
-    testCommand: 'node ../node_modules/.bin/truffle test `find test/*.js ! -name a_poly_oracle.js -and ! -name s_v130_to_v140_upgrade.js` --network coverage',
```

Comment:
` testCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle test `find test/*.js ! -name a_poly_oracle.js -and ! -name s_v130_to_v140_upgrade.js` --network coverage',`
`compileCommand: 'node --max-old-space-size=4096 ../node_modules/.bin/truffle compile --network coverage',`

Reply:
@maxsam4 maybe this will help, i am not 100% sure but it worth a try :)

Reply:
@nervehammer We need `find test/*.js ! -name a_poly_oracle.js -and ! -name s_v130_to_v140_upgrade.js`

`--max-old-space-size=4096` This however, might do the trick. Worth a try at the very least.